index.html

```diff
@@ -19,20 +19,94 @@
             position: absolute;
             top: 40px;
             left: 5px;
-            background-color: rgba(255, 255, 255, 0.5); 
+            border: 1px solid black;
+            border-radius: 4px;
+            padding: 5px;
+            background-color: rgba(255, 255, 255, 0.75);
+        }
+        .point-label {
+            position: absolute;
+            transform: translateY(-100%);
+            padding: 0px 5px;
+            border-radius: 9999px 9999px 9999px 0px;
+            color: white;
+            background-color: rgba(0, 0, 0, 0.5);
+            white-space: nowrap;
+        }
+        .crosshairs {
+            position: absolute;
+            transform: translate(-50%, -50%);
+        }
+        .crosshairs::before {
+            content: "";
+            position: absolute;
+            width: 12px;
+            height: 2px;
+            transform: translate(-50%, -50%);
+            background-color: red;
+        }
+        .crosshairs::after {
+            content: "";
+            position: absolute;
+            width: 2px;
+            height: 12px;
+            transform: translate(-50%, -50%);
+            background-color: red;
+        }
```

Comment:
For what it's worth, these crosshairs (at least as I've implemented them) are fairly expensive since they're rendered as two separate HTML elements, and the number of overlay elements seems to be a limiting factor for performance.